test/dyte/resources/organizations_test.rb

```diff
@@ -20,9 +20,19 @@ def test_update
     body = {name: "New and Better Name"}
     organization_id = "bbb0e958-93ce-41c7-9745-16e7aa51c568"
     stub_request(:patch, "#{Dyte::Client::BASE_URL}/orgs/#{organization_id}")
-      .to_return(body: File.new("test/fixtures/organizations/update.json"), headers: {content_type: "application/json"})
+      .to_return(body: File.new("test/fixtures/organizations/sample.json"), headers: {content_type: "application/json"})
```

Comment:
Was this intended?

Reply:
yes, since the payload was the same. However, I've changed it to separate files.